### PR TITLE
Remove duplicated imports in some documentation examples

### PR DESCRIPTION
**Pull request by M87K452b**
## Overview: What does this pull request change?

Removed duplicate import.

## Motivation and Explanation: Why and how do your changes improve the library?

Checked all the docstrings of the files in */manim* that makeup reference manual. Removed the only redundant import. The other duplicate mentioned in the issue is necessary for the decorator @animation_override. There were no other unnecessary imports found.

## Links to added or changed documentation pages
 

[Example Tree](https://docs.manim.community/en/stable/reference/manim.mobject.graph.Graph.html)

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by M87K452b** [comment]
I went through the references manual again checking for duplicates. 
The other multi-line imports are either for imports from multiple modules or 

of the following nature.

     from manim import *
     import manim

So, the changes proposed in the issue have been made.

File: manim/mobject/mobject.py [hunk 1]
```diff
@@ -2417,7 +2417,10 @@ def init_sizes(sizes, num, measures, name):
     def sort(self, point_to_num_func=lambda p: p[0], submob_func=None):
         """Sorts the list of :attr:`submobjects` by a function defined by ``submob_func``."""
         if submob_func is None:
-            submob_func = lambda m: point_to_num_func(m.get_center())
```
Comment (hydrobeam) [inline manim/mobject/mobject.py:2420]: **(question):** Is there a particular reason this was changed? If not, it should probably not be changed in this PR.

**Review by hydrobeam — APPROVED** [review]
LGTM, left a question about a change in syntax though🤔

File: manim/mobject/mobject.py (hunk 1 above)
Comment (M87K452b) [reply manim/mobject/mobject.py:2420]: I didn't change any code. I was only working with docstrings. I might have made a mistake while updating my local branch. @hydrobeam Please advise on how I go about fixing this. Thank you!

Comment (Darylgolden) [reply manim/mobject/mobject.py:2420]: I think precommit-ci did it

Comment (Darylgolden) [reply manim/mobject/mobject.py:2420]: Could you just manually undo the changes? That should fix it

**Review by Darylgolden — CHANGES_REQUESTED** [review]

File: manim/mobject/mobject.py [hunk 2]
```diff
@@ -2417,7 +2417,10 @@ def init_sizes(sizes, num, measures, name):
...
+
+            def submob_func(m):
+                return point_to_num_func(m.get_center())
+
```
Comment (hydrobeam) [inline manim/mobject/mobject.py:2423]:
```suggestion
            submob_func = lambda m: point_to_num_func(m.get_center())
```

Comment (M87K452b) [reply manim/mobject/mobject.py]: Thanks! I will do that!

**Comment by M87K452b** [comment]
@Darylgolden Hello, the change requested is already in the desired form of a  function instead of the lambda expression in the latest update.

**Review by Darylgolden — APPROVED** [review]
Thank you!

**Pull request merged**